docs/modules/REST.md

```diff
@@ -69,6 +69,16 @@ Adds HTTP authentication via username/password.
  * `[Part]` xml
 
 
+### amNTLMAuthenticated
+
+Adds Microsoft NTLM authentication via username/password.
+
+ * `param` $username
+ * `param` $password
+ * `[Part]` json
+ * `[Part]` xml
+
```

Comment:
This file is auto-generated from PHP file before release, so there is no need to edit it manually.